pkg/network/pod/annotations/generator.go

```diff
@@ -98,6 +102,37 @@ func (g Generator) GenerateFromSource(vmi *v1.VirtualMachineInstance, sourcePod
 	return annotations, nil
 }
 
+func (g Generator) GenerateFromPod(vmi *v1.VirtualMachineInstance, pod *k8Scorev1.Pod) (map[string]string, error) {
```

Comment:
Under this file, we have two similar methods: `GenerateFromSource` and `GenerateFromPod`.
The name is also confusing: it mentions "from pod", but it is "from vmi and pod".

Perhaps `GenerateFromActive` or `GenerateFromTarget` is a better option.
A description for the two will also help, clarifying that is the expected pod input.

Reply:
Renamed `GenerateFromPod` to `GenerateFromActivePod`.